browser/components/loop/content/css/panel.css

```diff
@@ -551,11 +578,11 @@ html[dir="rtl"] .room-entry-call-btn {
 
 html[dir="rtl"] .room-entry-context-actions > .dropdown-menu {
   right: auto;
-  left: 21px;
+  left: 45px;
 }
 
 .room-entry-context-actions > .dropdown-menu {
-  right: 21px;
+  right: 45px;
```

Comment:
Please make these two things part of the other spin-out bug: why is this 45px?  should the word context be changed or gotten rid of (it seems wrong).